src/scripts/main.js

```diff
@@ -1,3 +1,342 @@
 'use strict';
 
-// write code here
+const table = document.querySelector('table');
+const headers = table.querySelectorAll('th');
+const tableBody = table.querySelector('tbody');
+
+headers.forEach(function(header) {
+  header.dataset.sorted = 'DESC';
+});
+
+function sortColumn(index, isASC) {
+  let sortedRows = [];
+
+  if (isASC) {
+    if (index === 4) {
+      sortedRows = Array.from(tableBody.rows).sort((a, b) =>
+        (salaryToNumber(a.cells[index].innerHTML)
+        - salaryToNumber(b.cells[index].innerHTML)));
+    } else {
+      sortedRows = Array.from(tableBody.rows).sort((a, b) =>
+        (a.cells[index].innerHTML.localeCompare(b.cells[index].innerHTML)));
+    }
+  } else {
+    if (index === 4) {
```

Comment:
Try to avoid nesting if...else. Please rewrite this part without it. You can use a ternary operator for that